ansible/roles/create_user/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Include encrypted variables
+  include_vars:
+    file: vars/vault.yml
+
+- name: Create user authz group
+  group:
+    name: authz
+    state: present
+  become: true
+  become_user: root
+
+- name: Create user authz
+  user:
+    name: authz
+    password: "{{ authz_password }}"
```

Comment:
i dont think we need to give this user a password + does it need sudo permissions?

Reply:
Fix. Removed the password and removed the sudo group for the user.